mmv1/products/tags/api.yaml

```diff
@@ -93,3 +93,59 @@ objects:
 
           A timestamp in RFC3339 UTC "Zulu" format, with nanosecond resolution and up to nine fractional digits. Examples: "2014-10-02T15:01:23Z" and "2014-10-02T15:01:23.045123456Z".
         output: true
+  - !ruby/object:Api::Resource
+    name: 'TagValue'
+    min_version: beta
+    base_url: tagValues
+    self_link: "tagValues/{{name}}"
+    update_verb: :PATCH
+    update_mask: true
+    description: A TagValue is a child of a particular TagKey. TagValues are used to group cloud resources for the purpose of controlling them using policies.
+    references: !ruby/object:Api::Resource::ReferenceLinks
+      guides:
+        'Official Documentation':
+          'https://cloud.google.com/resource-manager/docs/tags/tags-creating-and-managing'
+      api: 'https://cloud.google.com/resource-manager/reference/rest/v3/tagValues'
+    properties:
+      - !ruby/object:Api::Type::String
+        name: name
+        description: |
+          The generated numeric id for the TagValue.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: parent
+        description: |
+          Input only. The resource name of the new TagValue's parent. Must be of the form tagKeys/{tag_key_id}.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::String
+        name: shortName
+        description: |
+          Input only. User-assigned short name for TagValue. The short name should be unique for TagValues within the same parent TagKey.
+
+          The short name must be 63 characters or less, beginning and ending with an alphanumeric character ([a-z0-9A-Z]) with dashes (-), underscores (_), dots (.), and alphanumerics between.
+        input: true
+        required: true
+      - !ruby/object:Api::Type::String
+        name: namespacedName
+        description: |
+          Output only. Namespaced name of the TagValue. Will be in the format {organizationId}/{tag_key_short_name}/{shortName}.
+        output: true
+      - !ruby/object:Api::Type::String
+        name: description
+        description: |
+          User-assigned description of the TagValue. Must not exceed 256 characters.
```

Comment:
Optional:
we can enforce this with the terraform.yaml override
```
validation: !ruby/object:Provider::Terraform::Validation
  function: 'validation.StringLenBetween(0, 256)'
```

Reply:
Changed in ce9242e - also added validation for short name (and for both fields on tagkey)